### PR TITLE
data base fix for ordering,referring and contact Physicians on fee sheet

### PR DESCRIPTION
**Pull request by teryhill**
data base fix for ordering,referring and contact Physicians on fee sheet

### CODE REVIEW

File: sql/database.sql [hunk 1]
```diff
@@ -1281,10 +1281,14 @@ CREATE TABLE `form_encounter` (
   `stmt_count`        int  NOT NULL DEFAULT 0,
   `provider_id` INT(11) DEFAULT '0' COMMENT 'default and main provider for this visit',
   `supervisor_id` INT(11) DEFAULT '0' COMMENT 'supervising provider, if any, for this visit',
+  `ordering_physician` INT(11) DEFAULT '0' COMMENT 'ordering provider , if any, for this visit',
+  `referring_physician` INT(11) DEFAULT '0' COMMENT 'referring provider, if any, for this visit',
+  `contract_physician` INT(11) DEFAULT '0' COMMENT 'contract provider, if any, for this visit',
```
Comment (aethelwulffe) [inline sql/database.sql]: Did we decide to call this field x_physician vs. x_clinician?
Otherwise, this looks good.

File: sql/database.sql [hunk 2]
```diff
@@ -1281,10 +1281,14 @@ CREATE TABLE `form_encounter` (
...
   `provider_id` INT(11) DEFAULT '0' COMMENT 'default and main provider for this visit',
   `supervisor_id` INT(11) DEFAULT '0' COMMENT 'supervising provider, if any, for this visit',
+  `ordering_physician` INT(11) DEFAULT '0' COMMENT 'ordering provider , if any, for this visit',
+  `referring_physician` INT(11) DEFAULT '0' COMMENT 'referring provider, if any, for this visit',
+  `contract_physician` INT(11) DEFAULT '0' COMMENT 'contract provider, if any, for this visit',  
   `invoice_refno` varchar(31) NOT NULL DEFAULT '',
   `referral_source` varchar(31) NOT NULL DEFAULT '',
   `billing_facility` INT(11) NOT NULL DEFAULT 0,
   `external_id` VARCHAR(20) DEFAULT NULL,
+  `pos_code` tinyint(4) default NULL,
```
Comment (aethelwulffe) [inline sql/database.sql]: The facility code should come from "service_facility" and not a separate POS code.  That means if you have an error in the facility info, you can change that, and all the encounters are fixed.

Comment (aethelwulffe) [inline sql/database.sql]: Make that "facility_id" and not "service_facility".  The two of these should be clearly marked in the encounter form to indicate that the facility_id field is specifically "Place of Service" and the "Billing Facility" is also clearly marked.  I believe each already has the code to list only the correct list of facilities for each dropdown.

Comment (teryhill) [inline sql/database.sql]: removed offending field. Not in the LibreHealth code anyway

File: sql/database.sql (hunk 1 above)
Comment (teryhill) [inline sql/database.sql]: This is just a fix to get the fields for the code that has already been placed into the code base into the table

**Comment by aethelwulffe** [comment]
I suppose there was no issue with you needing the POS code in the encounter.  Since the other fields are already there...and they won't kill anyone, I will approve this.

**Pull request merged**